packages/laconia-event/src/DynamoDbStreamRecord.js

```diff
@@ -1,18 +1,36 @@
-const AWS = require("aws-sdk");
+require("aws-sdk/clients/dynamodb");
+const converter = require("aws-sdk/lib/dynamodb/converter");
 
 module.exports = class DynamoDbStreamRecord {
   constructor(data) {
     this.data = data;
   }
 
   get jsonNewImage() {
-    return AWS.DynamoDB.Converter.unmarshall(this.newImage);
+    return converter.unmarshall(this.newImage);
   }
 
   get newImage() {
     return this.data.NewImage;
   }
 
+  get parseRecord() {
+    return {
+      eventName: this.eventName,
```

Comment:
Where this come from?

Reply:
I added an additional getter as I was just spiking around. I haven't used a getter in is before ( cool stuff!)

Looking at it now the name doesn't seem so thoughtful. Maybe just record would be more appropriate

Reply:
You did not set any value as eventName :)